Controllers/DishesController.js

```diff
@@ -0,0 +1,49 @@
+const DbService = require('../Services/DbService')
+const DishesService = require('../Services/DishesService')
+const JSONResponseService = require('../Services/JSONResponseService')
+
+let getAllDishes = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        try {
+            let dishes = await DishesService.getAllDishes(db, req)
+            let response = JSONResponseService.generateSuccessResponse()
+            response.message = "Requested starters retrieved successfully."
+            response.data = dishes
+            res.json(response)
+        } catch (e) {
+            let response = JSONResponseService.generateFailureResponse()
+            response.message = "The resources requested do not exist at the desired location."
+            res.json(response)
+        }
+    })
+}
+
+let getAllDishesOfType = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        if (
+            req.params.course === 'starters' ||
+            req.params.course === 'mains' ||
+            req.params.course === 'desserts' ||
+            req.params.course === 'refreshments'
+        ) {
+            try {
+                let dishes = await DishesService.getAllDishesOfType(db, req)
+                let response = JSONResponseService.generateSuccessResponse()
+                response.message = "Requested " + req.params.course + " retrieved successfully."
+                response.data = dishes
+                res.json(response)
+            } catch (e) {
+                let response = JSONResponseService.generateFailureResponse()
+                response.message = "The resources requested do not exist at the desired location."
+                res.json(response)
+            }
+        } else {
+            let response = JSONResponseService.generateFailureResponse()
+            response.message = "The resources requested do not exist at the desired location."
```

Comment:
A more specific error would be helpful here, just to say you must provide a valid category or something like that